apps/base-docs/tutorials/docs/6_subquery.md

```diff
@@ -0,0 +1,364 @@
+---
+title: 'Indexing Data with SubQuery'
+slug: /subquery
+description: A tutorial explaining SubQuery and providing swift guidance on configuring a project
+author: subquery
+keywords:
+  [
+    indexers,
+    RPC,
+    API,
+    graphql,
+    frontend,
+    smart contract development,
+    EVM,
+    Base,
+    Base network,
+    Base node providers,
+    Base providers,
+    blockchain development,
+    dApps,
+    smart contracts,
+    blockchain API
+  ]
+tags: ['smart contracts', 'api']
+difficulty: beginner
+hide_table_of_contents: false
+displayed_sidebar: null
+---
+
+## Objectives
+
+- Present the concept of indexers
```

Comment:
Please switch to Bloom's Taxonomy verb learning objectives.  What should the learner specifically be able to do when they complete this lesson?

- Configure SubQuery indexers to efficiently retrieve data from a smart contract